projects/templates/projects/issues/issue_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "../base.html" %}
+{% load projects_extras %}
+{% block title %}TAIPAN | Issue Edit{% endblock %}
+{% block content %}
+    {% if form.instance.id %}
+    <h1>Edit Issue</h1>
+    {% else %}
+    <h1>New Issue</h1>
+    {% endif %}
+    {% include 'projects/form-template.html' %}
+    {% if form.instance.id %}
+    <form action="{% url 'projects:delete_issue' form.instance.id form.instance.build_test.build.id %}" method="post">
+        {% csrf_token %}
+        <button type="submit" class="danger-button right" onclick="return confirm('Are you sure you want to delete?')">Delete</button>
```

Comment:
If you want a _really_ nice confirm, check out the [jquery ones](https://craftpip.github.io/jquery-confirm/). Alert and confirm! Protec and attac!